ui/app/src/components/PreviewComponentsPanel/PreviewComponentsPanel.tsx

```diff
@@ -78,13 +78,7 @@ export default function PreviewComponentsPanel() {
         }}
         withEmptyStateProps={{
           emptyStateProps: {
-            title: <FormattedMessage id="componentsPanel.emptyStateMessage" defaultMessage="No components found" />,
-            subtitle: (
-              <FormattedMessage
-                id="componentsPanel.emptyStateMessageSubtitle"
-                defaultMessage="Communicate with your developers to create the required components in the system."
-              />
-            )
+            title: <FormattedMessage id="componentsPanel.emptyStateMessage" defaultMessage="No contentTypes found" />
```

Comment:
- Why did you remove the subtitle?
- `contentTypes` is 2 words: "No content types found". We aren't listing content types here, are we? Think the original message was the right one. It's components that we're listing.